torch_optimizer/lookahead.py

```diff
@@ -75,7 +75,7 @@ def step(self, closure: OptLossClosure = None) -> OptFloat:
         for group in self.param_groups:
             if group['counter'] == 0:
                 self._update(group)
-            group['counter'] = (group['counter'] + 1) % self.k
```

Comment:
Can you please elaborate on this change?

Reply:
I forgot I included this but to me it looks like the first part can be carried out in-place and didn't change anything in my model training. Though, it may be better to instead do:
group[counter] += 1
group[counter] %= self.k

To make sure the modulo is over entire group and not 1. I can change back or convert to new form just let me know. 

The main item is the zero_grad(set_to_none) which saves me memory when training.